packages/tinyfiledialogs/tinyfiledialogs.1.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings to tinyfiledialogs"
+description: "Simple, cross-platform file dialogs"
+maintainer: ["Charles Averill <charlesaverill20@gmail.com>"]
+authors: ["Charles Averill <charlesaverill20@gmail.com>"]
+license: "MIT"
+tags: [
+  "GUI"
+  "Dialogs"
+  "File Dialog"
+  "Color Picker"
+  "Cross-Platform"
+  "TinyFileDialogs"
+  "Open File Dialog"
+  "Save File Dialog"
+  "Select Folder"
+  "Message Box"
+  "Input Box"
+]
+homepage: "https://github.com/CharlesAverill/tinyfiledialogs"
+doc: "https://charlesaverill.github.io/tinyfiledialogs/tinyfiledialogs/"
+bug-reports: "https://github.com/CharlesAverill/tinyfiledialogs/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.19"}
+  "ctypes"
+  "ctypes-foreign"
```

Comment:
```suggestion
  "ctypes" {>= "0.14.0"}
  "ctypes-foreign" {>= "0.14.0"}
```
This should take care of the lower bounds failures `failed: Library "ctypes-foreign" not found` where a too old `ctypes-foreign` is combined with a more recent `ctypes`. (I recently fixed something similar in #28372)

Reply:
Fixed, thank you!